Userland/Libraries/LibCore/UDPServer.cpp

```diff
@@ -61,15 +62,13 @@ bool UDPServer::bind(IPv4Address const& address, u16 port)
     return true;
 }
 
-ByteBuffer UDPServer::receive(size_t size, sockaddr_in& in)
+ErrorOr<ByteBuffer> UDPServer::receive(size_t size, sockaddr_in& in)
 {
-    // FIXME: Handle possible OOM situation.
-    auto buf = ByteBuffer::create_uninitialized(size).release_value_but_fixme_should_propagate_errors();
+    auto buf = TRY(ByteBuffer::create_uninitialized(size));
     socklen_t in_len = sizeof(in);
     ssize_t rlen = ::recvfrom(m_fd, buf.data(), size, 0, (sockaddr*)&in, &in_len);
     if (rlen < 0) {
-        dbgln("recvfrom: {}", strerror(errno));
-        return {};
+        return Error::from_errno(errno);
     }
```

Comment:
Can you use `Core::System::recvfrom()` instead?

Reply:
Yes, of course :^)